frontend/src/data/config/DatasetMetadata.ts

```diff
@@ -83,23 +83,48 @@ export const datasetMetadataList: DatasetMetadata[] = [
     update_time: "April 2021",
   },
   {
-    id: "acs_health_insurance-health_insurance_by_sex_age_county",
-    name: "Health insurance by sex, age and county",
+    id: "acs_health_insurance-by_age_county_processed",
+    name: "Health Insurance by age and county",
```

Comment:
I've been trying to slowly standardize everything on title casing unless it's a term that we put in bold or quotes or it's a proper noun; so i think these should be `Health insurance`